u5-rest-apis/app.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express')
+const { json, urlencoded } = require('body-parser')
+const { readBooks, readBook, createBook, replaceBook, updateBook, deleteBook, readCovidRecords, readCovidRecord, createCovidRecord, UpdateCovidRecord, DeleteCovidRecord } = require('./features/book-controller')
+
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+app.use(json())
+app.use(urlencoded({ extended: false }))
+
+// Routes
+app.get('/books', readBooks)
+app.get('/books/:isbn13', readBook)
+app.post('/books', createBook)
+app.put('/books/:isbn13', replaceBook)
+app.patch('/books/:isbn13', updateBook)
+app.delete('/books/:isbn13', deleteBook)
+
+// Routes Lab
+app.get('/covid', readCovidRecords)
+app.get('/covid/:stateId', readCovidRecord)
+app.post('/covid/', createCovidRecord)
+app.patch('/covid/:stateId', UpdateCovidRecord)
+app.delete('/covid/:stateId', DeleteCovidRecord)
+// TODO later
```

Comment:
ใน JS ตัวแปรควรขึ้นต้นด้วยตัวพิมพ์เล็กเสมอ คุณจะเห็นได้ว่า GitHub ทำสีผิดด้วยเหตุนี้!
In JS, variables should always start with a lowercase letter. You can see that GitHub even colors it wrong because of this!
```
updateCovidRecord
deleteCovidRecord
```
คุณยังสามารถลบ `// TODO` ได้เพราะคุณทำไปแล้ว!
You can also delete the `// TODO` because you've done it already!